docs/advanced/voronoi_sampling.rst

```diff
@@ -0,0 +1,85 @@
+.. _voronoi_sampling:
+
+=========================
+Sampling in Voronoi grids
+=========================
+
+In RT computations, the need to uniformly sample points in the cells of a grid often arises. A typical example
+is that of the evaluation of the average value of a function (e.g., a density function) within a cell.
+
+Whereas this task is relatively simple in regular grids, the situation is more complicated in Voronoi grids.
+Cells in :ref:`Voronoi grids <voronoi_grid>` are indeed convex polyhedra with an arbitrary number of faces,
+whose shapes and volumes are determined by the intial distribution of sites.
+
+The Voronoi helper class in Hyperion contains support for producing a list of sampling points for each cell.
+This is a simple example of its usage.
+
+We first initialise a cubic domain with unitary edge, and we fill it with 1E5 randomly-placed points:
+
+>>> import random, numpy as np
+>>> sites_arr = np.array([(random.uniform(0, 1), random.uniform(0, 1), random.uniform(0, 1)) for i in range(0, 100000)])
```

Comment:
A clearer way to do this would be:

```
N = 100000
x = np.random.uniform(N)
y = np.random.uniform(N)
z = np.random.uniform(N)
sites_arr = np.array([x, y, z]).transpose()
```

More verbose, but should be *much* faster. You can also then remove the import of the 'random' module.

A couple of other stylistic points (which you can ignore since they will no longer be relevant with my proposed changes):

* Only import one package per line (PEP8)
* You don't need the '0' in range since it will default to that, so ``range(100000)`` would give the same